setup.sh

```diff
@@ -8,6 +8,7 @@ erlc +debug_info -o src/main/resources/erlyberly/beam/ src/main/resources/erlybe
 
 # install floaty field
 git clone https://github.com/andytill/floaty-field.git
+$(cd floaty-field && git checkout 0048c66)
```

Comment:
Is a commit hash here because there are no tags?  If so, I can create a tag on floaty-field and jinterface and use that instead.